packages-urls/manager-install-rpm-release.yaml

```diff
@@ -1,2 +1,2 @@
-http://repository.cloudifysource.org/cloudify/20.03.18/community-release/cloudify-manager-install-community-20.03.18.rpm
+http://repository.cloudifysource.org/cloudify/20.03.18/release/cloudify-manager-install-community-20.03.18.rpm
```

Comment:
is the rpm name is the same name generated from the build ? I think it contains `x86_64` ?
@warolv

Reply:
@mabuaisha right, it's
cloudify-manager-install-20.03.18-community.x86_64.rpm